tests/unit/conditions/test_ecdsa_condition.py

```diff
@@ -134,20 +139,142 @@ def test_ecdsa_condition_verify_invalid_signature():
     invalid_signature = TEST_SIGNING_KEY.sign(
         b"Different message",
         hashfunc=ECDSAVerificationCall._hash_func,
-        sigencode=sigencode_der,
-    )
-    invalid_signature_b64 = base64.b64encode(invalid_signature).decode("utf-8")
+        sigencode=sigencode_string,
+    ).hex()
 
     condition = ECDSACondition(
         message=":message_variable",
         signature=":signature_variable",
-        verifying_key=TEST_VERIFYING_KEY_PEM,
+        verifying_key=TEST_VERIFYING_KEY_HEX,
     )
 
     context = {
         ":message_variable": TEST_MESSAGE,
-        ":signature_variable": invalid_signature_b64,
+        ":signature_variable": invalid_signature,
     }
     success, result = condition.verify(**context)
     assert not success
     assert result is False
+
+
+def test_ecdsa_condition_different_curves():
```

Comment:
Is it possible to parameterize this test to take (curve name, <other things>) tuples to simplify the test?

Similar comment for some of the integration tests?

Reply:
Follow-up in another PR to keep momentum.